pkg/reconciler/delivery/delivery.go

```diff
@@ -40,14 +41,23 @@ const (
 // Reconciler implements controller.Reconciler
 type Reconciler struct {
 	client      clientset.Interface
-	routeLister listers.RouteLister 
+	routeLister listers.RouteLister
+	followup    func(*v1.Configuration, time.Duration)
 }
 
 // Check that our Reconciler implements ksvcreconciler.Interface
 var _ configurationreconciler.Interface = (*Reconciler)(nil)
+
 var (
 	newPercent int64 = 50
 	oldPercent int64 = 50
+	// we use a global variable for now because we assume for simplicity that all Configurations
+	// use the same policy; in the future, we might want to associate a policy to each Configuration
+	policy Policy = Policy{
+		Mode: "time",
+		Percents: []int{0, 10, 50, 90},
+		Threshold: 20,
```

Comment:
Why wouldn't threshold be duration?

Reply:
Because threshold can be interpreted in other ways, not just time. Although the policies we implement right now are all time-based, I kept threshold to be int just in case we need to accommodate other metrics in the future.